blocks.py

```diff
@@ -16,7 +19,7 @@ class CounterIntervalSimulator(
         IntervalTrigger,
         GeneratorBlock):
 
-    version = VersionProperty("1.2.1")
+    version = VersionProperty("1.3.1")
```

Comment:
As far as I can tell these simulators aren't changing. Why do they need a version bump?

Reply:
when i updated docs and looked at the diff i found some mismatches between version numbers, this one for example is 1.3.1 in spec.json and the block class, but not in blocks.py
https://github.com/nio-blocks/simulator/blob/master/spec.json#L3